Sources/FireKit/UserSessionProfileManager.swift

```diff
@@ -85,11 +88,22 @@ open class UserSessionProfileManager<T: Codable>: UserSessionManager {
         guard let user = user else { return } //TODO: Maybe trigger end session, etc
         self.endSession()
         if user.isAnonymous && isNew {
```

Comment:
burada bir iş var, sonraki if de aynı çünkü and its not like the user can override a specific case